crates/rspack_plugin_javascript/src/visitors/dependency/harmony_detection_scanner.rs

```diff
@@ -106,7 +116,13 @@ impl Visit for TopLevelAwaitScanner {
   fn visit_constructor(&mut self, _: &Constructor) {}
   fn visit_function(&mut self, _: &Function) {}
 
-  fn visit_await_expr(&mut self, _: &AwaitExpr) {
-    self.has_top_level_await = true;
+  fn visit_await_expr(&mut self, await_expr: &AwaitExpr) {
+    if self.top_level_await.is_none() {
+      self.top_level_await = Some(Span::new(
+        await_expr.span.span_lo(),
+        BytePos::from_u32(await_expr.arg.span_lo().0 - 1),
```

Comment:
What if there's more spaces between the keyword and expression, for example: `await                  <expr>`